torch_geometric/nn/aggr/multi.py

```diff
@@ -2,23 +2,46 @@
 
 import torch
 from torch import Tensor
+from torch.nn import Linear, MultiheadAttention
 
 from torch_geometric.nn.aggr import Aggregation
 from torch_geometric.nn.resolver import aggregation_resolver
 
 
 class MultiAggregation(Aggregation):
-    def __init__(self, aggrs: List[Union[Aggregation, str]],
-                 aggrs_kwargs: Optional[List[Dict[str, Any]]] = None):
+    r"""Performs aggregations with one or more aggregators and combines
+        aggregated results.
+
+    Args:
+        aggrs (list): The list of aggregation schemes to use.
+        aggrs_kwargs (list, optional): Arguments passed to the
+            respective aggregation function in case it gets automatically
+            resolved. (default: :obj:`None`)
+        mode (string, optional): The combine mode to use for combining
+            aggregated results from multiple aggregations (:obj:`"cat"`,
+            :obj:`"proj"`, :obj:`"sum"`, :obj:`"mean"`, :obj:`"max"`,
+            :obj:`"min"`, :obj:`"logsumexp"`, :obj:`"std"`, :obj:`"var"`,
+            :obj:`"attn"`). (default: :obj:`"cat"`)
+        mode_kwargs (dict, optional): Arguments passed for the combine `mode`.
```

Comment:
Let‘s document some options here such as in_channels, out_channems and num_heads